generate_models.sh

```diff
@@ -29,4 +29,4 @@ final as (
     from fields
 )
 
-select * from final" >> $1/models/$2__$5.sql
+select * from final" > $1/models/$2__$5.sql
```

Comment:
I think this one should actually stay as a `>>` since you've already replaced the "appending" `>` on line 4. On line 32, you _want_ it to add to the file made in line 4. 

I just tested this change to see if it's working as intended. Tests I ran:
1. [x] Test if it still populates the macros & model/tmp + model directories 
2. [x] Test if it overwrites the original files:
- It looked like running the respective `source` code that is generated with the `dbt run` command only worked to overwrite the macros IF the models/tmp didn't exist OR
- It requires the .yml files to be in the `models` directory before overwriting.

Did you experience the same observations when you tested the code? I'd recommend adding in a quick blurb in the macro itself for expected behavior for overwriting.